packages/typespec-test/test/loadtesting_modular/generated/typespec-ts/src/loadTestAdministration/api/operations.ts

```diff
@@ -0,0 +1,1058 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  Test,
+  FileInfo,
+  TestAppComponents,
+  AppComponent,
+  TestServerMetricConfig,
+  FileInfoList,
+  TestsList,
+} from "../models/models.js";
+import {
+  isUnexpected,
+  AzureLoadTestingContext as Client,
+  LoadTestAdministrationCreateOrUpdateAppComponents200Response,
```

Comment:
The RLC layer response type name is weird because we have enabled operation group as the response type prefix to resolve the RLC layer operation name conflicts.